client/sugus/src/routes/wizard/continue/wizard-continue.tsx

```diff
@@ -66,7 +66,7 @@ function WizardContinue(props: ConnectionProps) {
                     if (answer.key !== question.key) {
                         return { ...answer };
                     }
-                    return { ...answer, value: value };
+                    return { ...answer, value: value, internal: question.internal };
```

Comment:
```suggestion
                    return { ...answer, value: value, internal: question.internal as boolean };
```
non compila polo prettier